src/CsvParser.php

```diff
@@ -190,7 +197,14 @@ protected function row()
 
             if ($row !== false) {
                 if (count($this->header) != count($row)) {
-                    throw new \RuntimeException('Die Anzahl der Zeilen Felder stimmen nicht mit der Anzahl der Spaltenfelder Überein');
+                    throw new \RuntimeException(
+                        sprintf(
+                            'the number of columns on line #%d [%d] does not match those from the header line [%d]',
```

Comment:
the current exception message wasn't very helpful (+ in some weird language I have never seen)